transformers/integration_tests/osm_change_test.go

```diff
@@ -0,0 +1,68 @@
+package integration_tests
+
+import (
+	"strconv"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/makerdao/vdb-mcd-transformers/test_config"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/events/osm_change"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/test_data"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+	"github.com/makerdao/vulcanizedb/libraries/shared/fetcher"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+// TODO update when real event log exists
+var _ = XDescribe("OsmChange EventTransformer", func() {
```

Comment:
As you can see, I wasn't able to find any `change` events in Kovan. I _think_ it makes sense that none have happened so far, since the `change` event only gets called when the OSM contract wants to switch to using a different data-source. And that doesn't seem like something that would happen in the normal course of events, but lmk if I'm thinking about it wrong

Reply:
Makes sense to me but I think we probably should double check Mainnet - I think the integration tests are pointed there now, no?